lib/features/family/features/profiles/widgets/give_bottomsheet.dart

```diff
@@ -152,4 +171,18 @@ class GiveBottomSheet extends StatelessWidget {
       },
     );
   }
+
+  void showEmptyWalletBottomSheet(BuildContext context) {
+    context.pop();
+
+    showModalBottomSheet<void>(
```

Comment:
Except for the context.pop I would make the rest a static method in the EmptyWalletBottomSheet class, so that no matter where you show it, you show it exactly the same and don't have to continuously define the whole "showmodalbottomsheet" blabla. So then you can just do `EmptyWalletBottomSheet.show(context)`